docs/config/types.md

```diff
@@ -44,10 +44,6 @@ Suppose we are trying to send the health of a player over the remote. We may def
 
 <CodeBlock code="type Health = u8 (0..100)" />
 
```

Comment:
Might want to add an explaination as to _why_ it works
```suggestion

This works because limits are **inclusive**, and will include 0 and 100.
```